docs/framework/deployment/index.md

```diff
@@ -169,11 +169,11 @@ This article helps you get started deploying the .NET Framework with your applic
 |[Reducing System Restarts During .NET Framework 4.5 Installations](../../../docs/framework/deployment/reducing-system-restarts.md)|Describes the Restart Manager, which prevents reboots whenever possible, and explains how applications that install the .NET Framework can take advantage of it.|  
 |[Deployment Guide for Administrators](../../../docs/framework/deployment/guide-for-administrators.md)|Explains how a system administrator can deploy the .NET Framework and its system dependencies across a network by using System Center Configuration Manager (SCCM).|  
 |[Deployment Guide for Developers](../../../docs/framework/deployment/deployment-guide-for-developers.md)|Explains how developers can install .NET Framework on their users' computers with their applications.|  
-|[Deploying Applications, Services, and Components](../Topic/Deploying%20Applications,%20Services,%20and%20Components.md)|Discusses deployment options in Visual Studio, including instructions for publishing an application using the ClickOnce and Windows Installer technologies.|  
+|[Deploying Applications, Services, and Components](https://msdn.microsoft.com/library/wtzawcsz.aspx)|Discusses deployment options in Visual Studio, including instructions for publishing an application using the ClickOnce and Windows Installer technologies.| 
```

Comment:
The VS docs are also on docs. The URL is https://docs.microsoft.com/visualstudio/deploying-applications-services-and-components